vcd/datasource_vcd_network_isolated_v2.go

```diff
@@ -77,6 +77,29 @@ func datasourceVcdNetworkIsolatedV2() *schema.Resource {
 				Computed:    true,
 				Description: "Network prefix",
 			},
+			"dual_stack_enabled": {
+				Type:        schema.TypeBool,
+				Computed:    true,
+				Description: "Boolean value if Dual-Stack mode is enabled",
+			},
+			"secondary_gateway": {
+				Type:        schema.TypeString,
+				ForceNew:    true,
+				Computed:    true,
+				Description: "Secondary gateway (can only be IPv6 with dual_stack_enabled=true)",
+			},
+			"secondary_prefix_length": {
+				Type:        schema.TypeString,
+				ForceNew:    true,
+				Computed:    true,
+				Description: "Secondary prefix (can only be IPv6 with dual_stack_enabled=true)",
```

Comment:
If user makes a mistake, does the API throw a reasonable error message? In other words, we're fine here without additional validation?

Reply:
There is a validation that I put in place myself https://github.com/vmware/terraform-provider-vcd/pull/1071/files#diff-39f2208616694c9e642a0f12c92346cba46719384577cdc7988f096772ab5090R25-R31

User gets such error:
```
vcd_network_isolated_v2.ipv6-dualstack: Creating...
╷
│ Error: 'secondary_gateway' can only be IPv6 address
│ 
│   with vcd_network_isolated_v2.ipv6-dualstack,
│   on vcd.TestAccVcdIpv6Support.tf line 164, in resource "vcd_network_isolated_v2" "ipv6-dualstack":
│  164: resource "vcd_network_isolated_v2" "ipv6-dualstack" {
```

Reply:
Ah, I missed it, as I was looking for a ValidateFunc. Thanks!

Reply:
> Ah, I missed it, as I was looking for a ValidateFunc. Thanks!

This is a good idea. I should be able to actually do that. WIll attempt